OemPkg/BootMenu/BootMenu.c

```diff
@@ -550,13 +550,11 @@ BootMenuEntry (
                                  );
   if (mBootMenuPrivate.HiiHandle == NULL) {
     DEBUG ((DEBUG_ERROR, "%a: Error on HiiAddPackages. Code=%r\n", __FUNCTION__, Status));
-
-    return EFI_OUT_OF_RESOURCES;
+  } else {
+    Status = gBS->LocateProtocol (&gEdkiiFormBrowserEx2ProtocolGuid, NULL, (VOID **)&mBrowserEx2);

```

Comment:
Given there's an explicit DEPEX, it would be nice to at least have an ASSERT() on the status since it's expected to be successful.

Reply:
Added the ASSERT